data/strings/strings.txt

```diff
@@ -13383,44 +13383,44 @@
     zh-Hant = 樓層數（最大 %d）
 
   [error_enter_correct_storey_number]
-    comment = Error message in Editor when a user tries to set the number of floors for a building higher than 25
-    tags = android,ios
-    en = The number of floors must non exceed 25
-    ar = يجب ألا يتجاوز عدد طوابق المبنى 25
-    be = Колькасць паверхаў павінна быць не больш за 25
-    bg = Броят на етажите не трябва да надвишава 25
-    ca = El nombre de plantes no pot excedir de 25
-    cs = Opravte počet poschodí, max. 25
-    da = Rediger bygningen med maksimum 25 etager
-    de = Das Gebäude kann nicht mehr als 25 Etagen haben
-    el = Ο αριθμός των ορόφων δεν πρέπει να υπερβαίνει τους 25
-    es = El número de plantas no debe superar 25
-    et = Korruste arv ei või olla üle 25
-    eu = Landare kopurua ez da 25etik gorakoa izan behar
-    fa = تعداد طبقات نباید بیش از 25 باشد
-    fi = Muokkaa kerroksia, enintään 25 kerrosta
-    fr = Le nombre d'étages ne doit pas dépasser 25
-    hu = Legfeljebb 25 emeletes épület szerkesztése
-    id = Edit bangunan dengan maksimum 25 lantai
-    it = Il numero di piani non deve superare 25
-    ja = 最高25階までのビルを編集
-    ko = 최대 25층까지 입력하세요
-    mr = मजल्यांची संख्या २५ पेक्षा जास्त नसावी
-    nb = Rediger bygningen med maks. 25 etasjer
-    nl = Bewerk het gebouw met maximaal 25 verdiepingen
-    pl = Edytuj budynek z maksymalną liczbą 25 pięter
-    pt = Editar o edifício com um máximo de 25 pisos
-    pt-BR = O número de andares não pode ultrapassar 25
-    ro = Numărul de etaje nu trebuie să depășească 25
-    ru = Количество этажей не должно превышать 25
-    sk = Upraviť s maximálne 25 poschodiami
-    sv = Redigera byggnaden med max 25 våningar
-    th = แก้ไขอาคารที่มีจำนวนชั้นมากกว่า 25
-    tr = Kat sayısı 25'i geçmemelidir
-    uk = Редагуйте будівлі висотою максимум 25 поверхів
-    vi = Sửa tòa nhà có tối đa 25 tầng
-    zh-Hans = 编辑最多 25 层的建筑
-    zh-Hant = 編輯最多 25 層的建築
+    comment = Error message in Editor when a user tries to set the number of floors for a building higher than %d floors
+    tags = android,ios
+    en = The number of floors must non exceed %d
```

Comment:
If we added int param to this string, we should update iOS code?

Reply:
Int param was already used there :)

Reply:
How it was already used if pervious string was without Int?